### PR TITLE
Internal trig advise user api

### PR DESCRIPTION
**Pull request by KPostOffice**
## Related Issues and Dependencies

Make internal trigger advise requests from investigator because Kebechet no longer sends advise requests unless specifically requested
## This introduces a breaking change

- [ ] Yes
- [ ] No

## This Pull Request implements

… Explain your changes.

### CODE REVIEW

**Comment by KPostOffice** [comment]
Fixes: #622

**Review by harshad16 — COMMENTED** [review]
lgtm
is this tested ? will it be possible to be tested in test cluster?

**Comment by KPostOffice** [comment]
@harshad16 Yeah, I tested this locally. Monitoring behavior in the test cluster is definitely a good idea as these are pretty large changes.

**Comment by codificat** [comment]
Unsure why the mypy check is failing, but hoping that #635 might fix it

**Comment by harshad16** [comment]
> Yeah, I tested this locally. Monitoring behavior in the test cluster is definitely a good idea as these are pretty large changes.

ack, will get this in test

File: thoth/investigator/common.py [hunk 1]
```diff
@@ -103,6 +108,7 @@ async def wait_for_limit(openshift: OpenShift, workflow_namespace: str):
         _LOGGER.debug("Current number pending = %d", total_pending)
 
 
+<<<<<<< HEAD
```
Comment (codificat) [inline thoth/investigator/common.py:111]: unresolved conflict

File: thoth/investigator/hash_mismatch/investigate_hash_mismatch.py [hunk 2]
```diff
@@ -65,6 +73,7 @@ async def parse_hash_mismatch(mismatch: Dict[str, Any], openshift: OpenShift, gr
             )
 
     if Configuration.THOTH_INVESTIGATOR_SCHEDULE_KEBECHET_ADMIN:
+<<<<<<< HEAD
```
Comment (codificat) [inline thoth/investigator/hash_mismatch/investigate_hash_mismatch.py:76]: unresolved conflict

File: thoth/investigator/missing_package/investigate_missing_package.py [hunk 3]
```diff
@@ -55,6 +59,7 @@ async def parse_missing_package(package: Dict[str, Any], openshift: OpenShift, g
         )
 
     if Configuration.THOTH_INVESTIGATOR_SCHEDULE_KEBECHET_ADMIN:
+<<<<<<< HEAD
```
Comment (codificat) [inline thoth/investigator/missing_package/investigate_missing_package.py:62]: unresolved conflict

**Review by codificat — CHANGES_REQUESTED** [review]
There are some unresolved conflicts that slipped in...

**Pull request closed without merging**